### PR TITLE
Document the composes and releases subcommands in man page.

### PR DESCRIPTION
**Pull request by bowlofeggs**
The high level overview in the man page was missing documentation
about the composes and releases subcommands.

Signed-off-by: Randy Barlow <randy@electronsweatshop.com>

### CODE REVIEW

File: docs/user/man_pages/bodhi.rst [hunk 1]
```diff
@@ -54,6 +54,10 @@ Commands
 There are two commands, ``overrides`` and ``updates``. They are described in more detail in their
 own sections below.
 
+``bodhi composes <subcommand> [options] [args]``
```
Comment (Zlopez) [inline docs/user/man_pages/bodhi.rst:61]: I don't see the composes command anywhere in this man page. There is section for `overrides`, `releases` and `updates`. But no section for `composes`.

**Review by Zlopez — CHANGES_REQUESTED** [review]
Missing the `composes` section, otherwise it looks good.

File: docs/user/man_pages/bodhi.rst (hunk 1 above)
Comment (bowlofeggs) [reply docs/user/man_pages/bodhi.rst:61]: Heh, so you are right. I thought I had done that before but apparently not. I'll write that up now.

Comment (bowlofeggs) [reply docs/user/man_pages/bodhi.rst:61]: @Zlopez I have added docs on the compose command now.

**Review by Zlopez — APPROVED** [review]

**Review by cverna — APPROVED** [review]
+1

**Comment by bowlofeggs** [comment]
The Rawhide failure is due to https://bugzilla.redhat.com/show_bug.cgi?id=1636611 and can be ignored.

**Comment by bowlofeggs** [comment]
This feature is planned for inclusion in the upcoming 3.11.0 release: https://github.com/fedora-infra/bodhi/pull/2709

**Comment by bowlofeggs** [comment]
Bodhi 3.11.0b5 has been deployed to staging:

https://bodhi.stg.fedoraproject.org

It is also available via Copr:

https://copr.fedorainfracloud.org/coprs/bowlofeggs/bodhi-pre-release

**Comment by bowlofeggs** [comment]
This feature is now released with 3.11.0.

**Pull request merged**